pkg/util/timeutil/ptp/ptp_clock_linux.go

```diff
@@ -79,5 +79,9 @@ func (p Clock) Now() time.Time {
 		panic(err)
 	}
 
-	return timeutil.Unix(int64(ts.tv_sec)*1e9, int64(ts.tv_nsec))
+	return timeutil.Unix(int64(ts.tv_sec), int64(ts.tv_nsec))
+}
+
+func realtime() Clock {
+	return Clock{clockDeviceID: uintptr(C.CLOCK_REALTIME)}
 }
```

Comment:
Move this into `ptp_clock_linux_test.go`, since it's basically a test helper?

Reply:
Done.

Reply:
Going to revert this, because:
```
gazelle: .../pkg/util/timeutil/ptp/ptp_clock_linux_test.go: warning: use of cgo in test not supported
```

And `dev generate` removes this test from BUILD files if it has cgo in it.